docs/src/views/Layout.tsx

```diff
@@ -0,0 +1,212 @@
+import * as React from 'react'
+import { Button, Flex, Header, Icon, Segment, Divider } from '@stardust-ui/react'
+
+import CodeSnippet from '../components/CodeSnippet'
+import ExampleSnippet from '../components/ExampleSnippet'
+import DocPage from '../components/DocPage'
+import GuidesNavigationFooter from '../components/GuidesNavigationFooter'
+
+const code = value => <code>{value}</code>
+
+const link = (content, href, isExternal = false) => (
+  <a href={href} {...isExternal && { target: 'blank' }}>
+    {content} {isExternal ? <Icon name="external" size="small" link fitted /> : ''}
+  </a>
+)
+
+const links = {
+  flex: link('Flex', '/components/flex'),
+  flexItem: link('Flex.Item', '/components/flex'),
+  grid: link('Grid', '/components/grid'),
+  segment: link('Segment', '/components/segment'),
+}
+
+export default () => (
+  <DocPage title="Layout">
+    <Header as="h2">Overview</Header>
+    <p>The following components are introduced by Stardust to handle layout aspects:</p>
+    <ul>
+      <li>{links.flex} - designed for layout in one dimension, either a row or a column.</li>
```

Comment:
seems that `Box` component, which is currently seen as primitive layout wrapper (0-D layout) should be also added to the scope here

Reply:
agreed to not advertise Box component as a way to handle layout aspects